sky/services/platform/system_chrome.mojom

```diff
@@ -70,5 +107,20 @@ interface SystemChrome {
   /// Platform Specific Notes:
   ///   If the overlay is unsupported on the platform, enabling or disabling
   ///   that overlay is a no-op and always return true.
-  SetEnabledSystemUIOverlays(uint32 overlays) => (bool success);
+  SetEnabledSystemUIOverlays(uint32 overlay_mask) => (bool success);
+
+  /// Specifies the style of the overlays that are visible on the embedder when
+  /// the applicatiomn is running.
+  ///
+  /// Arguments:
+  ///   style: A `SystemUIOverlayStyle` enum value that denotes the style
+  ///
+  /// Return value:
+  ///   boolean indicating if the preference was conveyed successfully to the
+  ///   embedder.
+  ///
+  /// Platform Specific Notes:
+  ///   If overlay style is unsupported on the platform, specifying a style is
+  ///   a no-op and always return true.
+  SetSystemUIOverlayStyle(SystemUIOverlayStyle style) => (bool success);
```

Comment:
Another approach would be a query function that returns what styles are available on the current platform. This kind of pattern emphasizes discoverability and helps avoid least-common-denominator problems (although in this case the number of supported styles is probably going to be pretty limited).